streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/YarnQueueServiceImpl.java

```diff
@@ -166,6 +168,40 @@ public void deleteYarnQueue(YarnQueue yarnQueue) {
     removeById(yarnQueue.getId());
   }
 
+  /**
+   * Only check the validation of queue-labelExpression when using yarn application or yarn-session
+   * mode or yarn-perjob mode.
+   *
+   * @param executionMode execution mode.
+   * @param queueLabel queueLabel expression.
+   */
+  @Override
+  public void checkQueueLabelFormatIfNeeded(ExecutionMode executionMode, String queueLabel) {
+    if (ExecutionMode.isYarnMode(executionMode)) {
+      ApiAlertException.throwIfFalse(isValid(queueLabel, true), ERR_FORMAT_HINTS);
```

Comment:
I suggest not to throw exception here, just return the result, it's up to the caller to decide whether it needs to throw an exception